### ISSUE TITLE
UglifyJsMinify - Terser Plugin Seems not working

### ISSUE THREAD

**Issue by Shashikant-Kew** [webpack-contrib/terser-webpack-plugin#609]
Webpack 5 version
Terser webpack plugin

### Bug report

UglifyJsMinify - Terser Plugin Seems not working from webpack

### Actual Behavior

Built files are not running properly. Crashing page.

### Expected Behavior

Plugin should minify and code files

### How Do We Reproduce?

Add Terser Plugin in webpack config
Minify option as UglifyJsMinify
Build on non prod server.
Access appliction.
Recieved files are giving error as
Uncaught Typeerror: can not read properties of undefined (reading bind)

**Comment by Shashikant-Kew** [comment]
https://stackoverflow.com/questions/78773698/terser-webpack-plugin-and-uglifyjs-webpack-plugin/78815126#78815126

**Comment by alexander-akait** [comment]
Please creare reprducible test repo, we have a lot of tests and eveyrything is fine

**Comment by Shashikant-Kew** [comment]
Yes but there was no change which would cause it in our repo. We retested reverting all change. Only when we remove plugin it works. Earlier it was fine. No issue. Only after 17/18 July It started giving errors.

**Comment by alexander-akait** [comment]
Please create reprducible test repo, I can't help you without more information
